joomla/joomla-cms
# [4.1] Do not set a media version in debug mode on assets

Pull Request for comment https://github.com/joomla/joomla-cms/issues/30444#issuecomment-709488213

### Summary of Changes
When the site runs in debug mode, then on every page load a new media version is added the the assets. This is problematic while debugging Javascript in a browser. Like that the browser always assumes that it is a new script and removes the breakpoints (at least this is the case in Firefox). If you do JS debugging, you know what I'm talking about ;-)

This behavior is a relict from the early days of Joomla (#1618) to simplify development. But nowadays every developer knows how to load the page with `Ctrl+F5` to refetch the assets.

CC: @dgrammatiko, @ceford 

### Testing Instructions
Open any page in debug mode.
Open the dev tools of the browser and check the loaded JS and CSS files.

### Actual result BEFORE applying this Pull Request
Assets are always loaded with a new media version after the path.

### Expected result AFTER applying this Pull Request
Assets are loaded without any media version.
